components/StoryCard.tsx

```diff
@@ -56,7 +81,12 @@ const StoryCard = ({ story, state, index, addFilter }: StoryCardParams): JSX.Ele
               width="250px"
               hoverable
               style={{ borderColor: borderColors[story.story_type] || 'gray' }}
-              onClick={() => dispatch(selectStory(story))}>
+              onClick={e => {
+                if (e.target.nodeName === 'TEXTAREA') {
+                  return;
+                }
```

Comment:
(not for this PR) we'll want a more reusable method to block modal opening on clicking on owner/tag filters too.